.github/workflows/base-ci-goreleaser.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
-          go-version: '~1.21.7'
+          go-version: '~1.22'
```

Comment:
We specify the patch version so should be `~1.22.1`

Reply:
@TylerHelmuth Fixed it now. Good to go?